superset-frontend/cccs-viz/cccs_export_dashboards.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+SOURCE_FILE=dashboards_export.json
+DEST_FILE=telemetry_dashboard.yaml
+DASHBOARDS="CCCS Telemery"
```

Comment:
Typo: Teleme>t<ry
```suggestion
DASHBOARDS="CCCS Telemetry"
```

Reply:
I'll fix that in the next pull request, I have a bunch of other changes on the go.